CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+### Added
+
+- [CL-1811] New Oostende verification method
+
```

Comment:
Please note that the release date is missing under `#Changelog`

It would be good to add:
```
- [CL-1558] Native surveys
```